src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -104,13 +105,15 @@ export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, Art
             artwork.cultural_maker &&
             this.renderArtistName(artwork.cultural_maker, null)}
         </Flex>
-        <Flex flexDirection="row">
-          <Serif italic color="black60" size="3t" m="0" p="0">
-            {artwork.title},{" "}
-          </Serif>
-          <Serif color="black60" size="3t" m="0" p="0">
-            {artwork.date}
+        <Flex flexDirection="row" flexWrap="wrap">
+          <Serif numberOfLines={1} ellipsizeMode="tail" italic color="black60" size="3t">
```

Comment:
Looks like this wasn't totally clear on the ticket, but we _do_ want to wrap the title to multiple lines if it's longer than one line! (see this clarification from Sam: https://artsy.slack.com/archives/C9YNS4X32/p1562692524406500?thread_ts=1562692268.405300&cid=C9YNS4X32)

Reply:
Oh I see, ok just updated.